http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -36,6 +36,11 @@
  * Static factory methods for producing common configurations of {@link FeignClientFactory}, which in turn may be used
  * to create HTTP proxies for HTTP remoting clients. The returned instances serialize server-side exceptions as JSON and
  * decode any 204 response as an {@link Optional#absent} in case the proxied interface is of type {@link Optional}.
+ * <p>
+ *     All factories take a User Agent and this will be embedded as the User Agent header for all requests.
+ *     For services, recommended user agents are of the form: ServiceName (Version), e.g. MyServer (1.2.3)
+ *     For services that run multiple instances, recommended user agents are of the form:
+ *       ServiceName/InstanceId (Version), e.g. MyServer/12 (1.2.3)
```

Comment:
Also might want to use `{@code ...}` to differentiate the recommendations stylistically...e.g. `{@code ServiceName (Version)}`